src/Controls/samples/Controls.Sample.UITests/Issues/Issue17694.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 17694, "Circle view not rotating from center", PlatformAffected.UWP)]
+	public class Issue17694 : TestContentPage
+	{
+		protected override void Init()
+		{
+			var circleView = new CircleView
+			{
+				HeightRequest = 50,
+				WidthRequest = 50,
+				Drawable = new CircleDraw()
+			};
+
+			var button = new Button()
+			{
+				AutomationId = "Spin",
+				Text = "Spin",
+			};
+			button.Clicked += async (s, e) => 
```

Comment:
The animation is nice to have humans check, but for CI maybe we can have a button so go to a specific rotation? And then capture that?

Reply:
Oops, yeah lemme change that to just set the `Rotation` property